how-to/onboarding-guide.md

```diff
@@ -1,81 +1,196 @@
+by the software review lead and the current editorial board.
+
+#### Adhoc Guest Editors
+
+Adhoc editors are editors with specific skill sets that are brought in to lead
+a single review. Examples of when there might be a need for an ad-hoc editor
+include:
+
+* If there is a conflict of interest between a package submitter and the
+  editorial team (e.g., a close colleague of everyone on the team)
+* If the editorial board is at capacity handling the current review load
+* If a very specialized skill set is needed (in a one-off type of situation)
+
+In this case, you may consider using our internal reviewer sign-up list to see
+if someone who signed up to be a reviewer might want to serve as an editor.
+
+### Experience Required to Be an Editor
+
+We prefer that editors have some experience with reviewing software. This
+experience could come from a previous review they worked on with pyOpenSci,
+rOpenSci, or JOSS.
+
+### Editorial Mentorship
+
+If a potential volunteer does not have prior software editorial experience, we
+offer a **mentorship process**. Editor mentorship is where someone with
+existing editorial experience mentors the new editor through their first
+review(s).
+
+### Recruiting New Editors
+
+Recruiting new editors and maintaining a sufficient and well-balanced
+editorial board is the responsibility of the
+[Software Review Lead](https://www.pyopensci.org/handbook/governance/structure.html#software-review-lead)
+with support and advice from the editorial board.
+
+:::{note}
+For the time being, the Software Review Lead role is being filled by the
+Executive Director with support from the Editor in Chief as we define our
+process and track the volume of submissions that we need to support. In the
+future, we will find someone with interest in leading peer review for
+pyOpenSci.
+:::
+
+## Where to Find New Editors
+
+Typically the Editor in Chief will work with the Peer Review Lead to find and
+onboard new editors in scientific topical areas where pyOpenSci has existing
+gaps.
+
+You might find good candidates to be an editor through:
+
+* Your professional network: you may know people in a specific domain that
+  might be a good fit to be an editor for pyOpenSci.
+* Our [pyOpenSci list of contributor pool](https://www.pyopensci.org/our-community/index.html#pyopensci-community-contributors) which includes:
+  * Package authors and maintainers who have already submitted a package for
+    review to pyOpenSci
+  * Reviewers who have already led reviews for pyOpenSci
+  * Past guest editors
+* Colleagues that you know who have reviewed for the Journal of Open Source
+  Software (JOSS) or rOpenSci who have Python expertise
+* Community members in the pyOpenSci Slack (please post a call in our
+  `#software-review` channel)
+
+When all of the above fails to return a good new editor candidate, you can find
+support from our pyOpenSci community manager who will post an open call on our
```

Comment:
```suggestion
support from our pyOpenSci Community Manager who will post an open call on our
```

Reply:
while it's _technically_ correct to keep job titles all lower case, it's culturally//stylistically preferred to have them capitalized